### DISCUSSION TITLE
Changing from Pycaret to AutoGluon

### DISCUSSION THREAD

**Discussion by celestinoxp** [autogluon/autogluon#4605] [General]
As everyone has seen, pycaret has come to an end... the team abandoned the project (personal and job changes). So it's time to move to a project that has constant maintenance and code updated with the latest packages...

I'm currently migrating my projects to Autogluon, from what I understand it's the most similar to Pycaret I've found (low code) and it seems to work very well.

What do you think about this? What is your opinion?

**Comment by hakan-77** [comment]
Hi @celestinoxp, where are you on the migration? I'm at a similar place and it would be great to share experiences.

**Reply by celestinoxp** [reply]
@hakan-77  Migrating from pycaret to autogluon is very easy!
If you take a look at the examples (https://auto.gluon.ai/stable/index.html) you will see that it is not very different from pycaret... if you have any questions you can ask

**Reply by hakan-77** [reply]
@celestinoxp looking at the samples, I thought the same:) 

But you know the devil can be at the details sometimes.. Were you using mlflow integration at pycaret? looks like that's missing in autogluon. And, have you migrated time-series from one to the other?

**Reply by shchur** [reply]
Please feel free to share your experience in this thread (or other issues/discussions on GitHub). Would be curious to learn if there is any functionality that we could add or improve in AutoGluon that was available in PyCaret.

**Comment by hakan-77** [comment]


**Reply by duncreg** [reply]
It doesn't look like it's actively maintained at all. That thread suggests that the original creator isn't maintaining it anymore but won't relinquish control and there's an attempt at a fork.
